### ISSUE TITLE
no new activity alert to agent when task collaborator replies via email.

### ISSUE THREAD

**Issue by sjxiong** [osTicket/osTicket#5558]
### Prerequisites

* [Yes ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [No] Do you have any errors in the PHP error log, or javascript console?
* [Yes] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [Yes] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

no new activity alert to the assigned agent when a task collaborator replies via email.

### Steps to Reproduce

1. Create a Task, assign it to agent A, collaborator B
2. Write a reply in the ticket system 
3. Collaborator B receives the reply in the email and answered it directly via email 
4. reply from Collaborator B is added to the task.
5. The agent is not receiving any notice and has no idea collaborator B replied

**Expected behavior:** When a reply from collaborator B is added to the task, a new activity alert email should be sent to the assigned agent.

**Actual behavior:** No new activity alert email.

extra: when collaborator B replies directly in the ticket system, there is a new activity alert email sent to the agent.
![image](https://user-images.githubusercontent.com/15185770/85123135-f8735c80-b227-11ea-8d1a-367ec3a4f564.png)

### Versions

Server Information
--
osTicket Version | v1.14.1 (f1e9e88)  Upgrade— v1.14.2 is available
Web Server Software | Apache/2
MySQL Version | 10.3.23
PHP Version | 7.3.19

**Comment by antiuser** [comment]
see here https://github.com/osTicket/osTicket/pull/5554

**Comment by JediKev** [comment]
@sjxiong 

This has been addressed with:
- #5554 

Cheers.

**Comment by sjxiong** [comment]
> @sjxiong
> 
> This has been addressed with:
> 
> * #5554
> 
> Cheers.

Thanks! I'm not a programmer myself. Does it mean if I ask our IT support to pull the current code from github, this fix is already included in the source code? so when we update our system with this version, this problem will be solved? is it included in this version v1.14.2?

**Comment by JediKev** [comment]
@sjxiong 

No, you will have to manually apply the changes until the pull is merged. Once it is merged then it will be in the codebase.

Cheers.
